### PR TITLE
Add dan's name pronunciation

### PR DESCRIPTION
**Pull request by dadlerj**
Should I use a phonetic alphabet, or is this sufficient? E.g. "dæn"?

### CODE REVIEW

**Comment by sqs** [comment]
Maybe a YouTube video (or audio file) of pronouncing names would work better instead of trying to write it? That's what I find most useful on Wikipedia for pronunciation.

**Comment by nicksnyder** [comment]
> Should I use a phonetic alphabet

No, I don't think that is helpful.

Audio link is probably ideal, but another effective tip is to say something like "rhymes with <a common word that people know how to pronounce>"

**Comment by dadlerj** [comment]
I'm going to go with @nicksnyder's suggestion here to get this done quickly, but I agree with @sqs's suggestion in genearl!

File: company/team/index.md [hunk 1]
```diff
@@ -149,6 +149,7 @@ To add yourself to this page, copy the following template, paste it at the end o
 
 ## Dan Adler (he/him)
 
+- First name is pronounced **dan**
```
Comment (dadlerj) [inline company/team/index.md:152]:
```suggestion
- First name is pronounced **dan** (rhymes with "can")
```

File: company/team/index.md [hunk 2]
```diff
@@ -149,6 +149,7 @@ To add yourself to this page, copy the following template, paste it at the end o
 
 ## Dan Adler (he/him)
 
+- First name is pronounced **dan** (rhymes with "can")
```
Comment (nicksnyder) [inline company/team/index.md:152]:
```suggestion
- “Dan” rhymes with "can"
```


The part not in parenthesis doesn’t seem to have any information content

Comment (dadlerj) [reply company/team/index.md:152]: Hm, agreed, but the thinking was that both of the pieces could have information content for other names (e.g. see how Noemi did her bullet on this). I suppose there's no need to maintain a super-consistent format here though?

Comment (nicksnyder) [reply company/team/index.md:152]: my personal opinion is that I think it is ok to not be super consistent here

**Comment by rvantonder** [comment]
I link to this on my personal website, which is what my university used to figure out how to pronounce people's names at graduation: https://www.name-coach.com/rijnard. You just record (and can add phonetic spelling). I believe it's free to create/use: https://www.name-coach.com/namebadge

**Comment by mercadon** [comment]
> Maybe a YouTube video (or audio file) of pronouncing names would work better instead of trying to write it? That's what I find most useful on Wikipedia for pronunciation.

I actually have not had good luck with Google/Youtube audio pronunciation of my own name and I can imagine there are a lot of names that might be difficult to have audio files/videos for. So I'm not sure if this would be the best way to be consistent.

**Comment by nicksnyder** [comment]
@mercadon the site that Rijnard linked to allows everyone to record their own name, which seems like a better solution than what I suggested.

**Comment by sqs** [comment]
@mercadon I mean recording yourself pronouncing it, not finding an existing video. @rvantonder’s idea sounds great.

**Comment by mercadon** [comment]
> I link to this on my personal website, which is what my university used to figure out how to pronounce people's names at graduation: https://www.name-coach.com/rijnard. You just record (and can add phonetic spelling). I believe it's free to create/use: https://www.name-coach.com/namebadge

Going to give this a try though, this seems perfect!

**Comment by sqs** [comment]
@dadlerj Do you still intend to merge this PR? If not, please close it. I'll close it in a couple days if I haven't heard back (I'm just trying to clean up our PR list).

**Comment by dadlerj** [comment]
I'll do this the way other teammates are now doing it at some point instead!

**Pull request closed without merging**